client/src/components/Register/MethodTile.scss

```diff
@@ -83,17 +87,28 @@
   &__unavailable {
     &-mask {
       background: $tile-mask-background;
-      height: 100%;
+      border-top-left-radius: $tile-border-radius;
+      border-top-right-radius: $tile-border-radius;
+      color: $tile-mask-colour;
       width: 100%;
       left: 0;
-      padding: $tile-padding;
+      padding: $tile-padding/2 $tile-padding;
       position: absolute;
       top: 0;
+      z-index: 1;
     }
 
     &-title {
       font-size: $font-size-h1;
-      margin-top: 0;
+      display: inline;
+
+      &::after {
```

Comment:
Can we add this to the translated string rather than introducing the `::after` please?

Reply:
I disagree. The injected piece of punctuation is relevant only in the context it's displayed, which is the very use case for CSS.
The markup remains the same as it was before, where this title is a heading, and both it and the following paragraph are displayed block. In this situation (should someone desire) it makes very little sense to have the colon present in the text node.

Reply:
What about languages where a colon isn't used?

Reply:
ok, fair comment.